python_modules/dagster/dagster/core/execution/context/output.py

```diff
@@ -459,6 +464,44 @@ def test_handle_output():
 
         return self._user_events
 
+    def add_metadata(self, metadata: Dict[str, Any]) -> None:
```

Comment:
Last question: should this be called something like `add_output_metadata`, or is that redundant / incorrect in some way? @alangenfeld or @OwenKephart might have thoughts.

Reply:
Hmm, tricky. Thoughts are:
* seems redundant since we are in `handle_output`
* but technically `AssetMaterialization`  can be yielded as well ? so maybe the distinction is warranted?
* consistency with op version has merit 

so i think i lean `add_output_metadata` but could be convinced either way

Reply:
I'm down for `add_output_metadata`. Will change to that.